### ISSUE TITLE
After network interruptions client finished sync but with some files and folders missing

### ISSUE THREAD

**Issue by fduraibi** [owncloud/client#3077]
My server (at home behind comcast connection :-1: ) syncs with two machines. I added a number of files (~50 file = ~500MB) to one client and it synced fine with the server. The next day I started my work computer and it started syncing, however, my home connection wasn't stable so the client would fail and resume ...etc until I saw a green check by the end of the day. But instead of having the new files downloaded to this machine, all the files got delete (i have about 13,500 files in 2300 folders = 29GB) except few files that I created the other day in one folder and one file in another folder which I didn't touch for ages. (I have no idea why it kept these files only)

Luckily, after I went home the files are still on the server, so I made a backup! (plus I have synced copy in my other machine)

Today, at work, it started syncing again and took few hours to go through the files and gave me the green check. So I did a diff with my backup and I found there are many missing files and folders that the client didn't sync from the server.

the log file doesn't show errors, even server log, and the sqlite db blacklist table is empty. Only 'metadata' has a long list of files/folders but not the missing files/folders.

This issue https://github.com/owncloud/client/issues/1949 is very similar to mine but I don't see any error messages in the log file.

Client: 1.8.0 (on Linux)
Server: 8.0.2

**Comment by fduraibi** [comment]
More about the file that survived, (it is even a useless temp file!)
This is the file name:
/ownCloud/workspace/Circuitz/piPower [RPi Power Management]/Code/AssemblerApp/Debug/AssemblerApp.tmp

Under 'Circuitz' I have about 25 folders with many subfolders and files all are not synced except the piPower... folder.
and even inside the 'piPower...' folder it only kept the sub folders that lead to that file and didn't sync any of the other folders or files. (didn't even sync the other files inside the last 'Debug' folder).

Also I just found out that two of the files which I created on the work machine didn't get synced back to the server!
Not sure if the info i provided is useful, but it is all I can find and hope it might lead to some conclusion.

**Comment by guruz** [comment]
This is unfortunately not enough info to concluce anything :|
Do you still have the client log?
Did you change anything in your server setup/storage/DB?

**Comment by madmoneymike5** [comment]
What operating system is your server running on? I had similar issues running my server on Windows 7 Home Premium. Then I learned that the developers are dropping Windows support altogether. While swearing heavily, I taught myself basic Linux/Ubuntu and got the server up and running. I haven't had a single sync issue since.

**Comment by fduraibi** [comment]
- About the log, i thought I had it for yesterday, but when I checked it again today I saw the time so that file covers only the last sync at the end of the day and it shows no error messages.
- No recent changes in the server, last change was upgrading to v8.0 which was a week ago.
- The server is running CentOS 6 32bit with php65 (from remi repo).

So what is the safest way to have the client sync everything again? delete the client db?
